allensdk/internal/brain_observatory/time_sync.py

```diff
@@ -18,8 +18,14 @@
 LONG_STIM_THRESHOLD = 0.2     # seconds
 MAX_MONITOR_DELAY = 0.07     # seconds
 
+"""
+These are the old legacy key pairs, I kept these in comments because
+I want a record of what we assumed the key pairs looked like at one
+point. This is for anyone that has to refactor key finding in the future
+to have some idea of what keys are supposed to exist in sync dataset line labels
+
 VERSION_1_KEYS = {
-    "photodiode": "stim_photodiode",
+    "photodiode": "photodiode",
```

Comment:
Why change these key definitions for VERSION_1_KEYS and VERSION_2_KEYS?  This is actually the reality of what the keys were at those times (i.e. was "stim_photodiode", then was "photodiode", and now we are going back to "stim_photodiode").  I think by  using POSSIBLE_KEY_PAIRS you are enabling the use of either key anyway?

Reply:
Yeah this was a change I introduced I thought, that's my mistake these weren't supposed to be changed. I am enabling the use of both and I ended up removing these!

Reply:
@wbwakeman should resolve this conversation